option.go

```diff
@@ -13,26 +13,30 @@ type ClientOptions struct {
 	Host               string
 	User               string
 	Password           string
-	DBName             string
+	Database           string
 	SSLMode            string
 	Timezone           string
 	MaxOpenConnections int
 	MaxIdleConnections int
 	WithCache          bool
 	SavepointEnabled   bool
 	Logger             Logger
+	ApplicationName    string
+	ConnectTimeout     int
 }
 
 func (e ClientOptions) String() string {
-	return fmt.Sprintf("%s://%s:%s@%s:%d/%s?sslmode=%s;timezone=%s",
+	return fmt.Sprintf("%s://%s:%s@%s:%d/%s?sslmode=%s&timezone=%s&application_name=%s&connect_timeout=%d",
 		ClientDriver,
 		url.QueryEscape(e.User),
 		url.QueryEscape(e.Password),
 		e.Host,
 		e.Port,
-		e.DBName,
+		e.Database,
 		e.SSLMode,
 		e.Timezone,
+		e.ApplicationName,
+		e.ConnectTimeout,
```

Comment:
these parameters should be optional to avoid breaking compatibility with the old options

Reply:
They are: if the field is empty, it's ignored by postgres. However, I can pass them only if they are defined if you prefer.

Reply:
It's better I think it will be more explicit